content/docs/refs-and-the-dom.md

```diff
@@ -97,21 +138,24 @@ class CustomTextInput extends React.Component {
 }
 ```
 
-### Refs and Functional Components
+#### Refs and Functional Components
 
 **You may not use the `ref` attribute on functional components** because they don't have instances:
 
-```javascript{1,7}
+```javascript{1,8,13}
 function MyFunctionalComponent() {
   return <input />;
 }
 
 class Parent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.textInput = React.createRef();
+  }
   render() {
     // This will *not* work!
```

Comment:
Unrelated to this diff: I wonder if we should highlight this (comment) line too?